nixos/modules/services/web-apps/snipe-it.nix

```
@@ -454,25 +454,43 @@ in {
 
           # migrate db
           ${pkgs.php}/bin/php artisan migrate --force
+
+          # A placeholder file for invalid barcodes
+          invalid_barcode_location="${cfg.dataDir}/public/uploads/barcodes/invalid_barcode.gif"
+          [ ! -f "$invalid_barcode_location" ] \
```

```suggestion
          [ ! -e "$invalid_barcode_location" ] \
```
Just to be safe to not overwrite symlinks